pennylane/kernels/cost_functions.py

```diff
@@ -68,7 +68,7 @@ def polarity(
 
     .. code-block :: python
 
-        dev = qml.device('default.qubit')
+        dev = qml.device('default.qubit', wires=2)
```

Comment:
So we need this because we're hitting `dev.wires` later on?

Maybe we just switch `dev.wires` to be something explicit?

Reply:
Excactly. Could do that as well, but I found this a neat way, and it's not showcased much in the docs I believe, so maybe cool to have it somewhere?